### ISSUE TITLE
Player Name Change Doesn't Update on Revisions

### ISSUE THREAD

**Issue by sovrynn** [OvercastNetwork/Issues#2709]
https://oc.tc/revisions/docs

Jexel's name currently does not link anywhere, because he changed his name to bucko511 or something like that. 

edit: same for larrowhead and cowinkkeydinkinc, however these guys didn't change their usernames from the original so that's weird

**Comment by Pugzy** [comment]
Names on the revision page need to be linked an MC account manually, this is done for all internal staff members once they get the position. For OpenSource repositories such as rotations and docs anyone can contribute so no MC -> Github account link is created.

**Comment by jexeI** [comment]
Yes, the revisions page was showing my GITHUB username, and not my minecraft username, therefore, it doesn't use UUID to update my name.
